doc/nrf/device_guides/working_with_nrf/nrf70/developing/index.rst

```diff
@@ -3,17 +3,12 @@
 Developing with nRF70 Series
 ############################
 
-The |NCS| provides support for development on the nRF70 Series, using the nRF7002 DK (PCA10143) that includes the Wi-Fi® companion IC.
+The |NCS| provides support for developing networking applications with Wi-Fi connectivity using the nRF70 Series companion IC devices.
+Applications can be developed on the nRF7002 DK (PCA10143), which includes the nRF7002 companion IC, or on boards compatible with the nRF7002 EK (PCA63556) or the nRF7002 EB (PCA63561).
```

Comment:
This is fine, but it is going to be identical to the new Chapter entry sentence ("Working with nRF70 Series"). No problem with me, repeating it here, just pointing it out to you @richabp .

Maybe, though, add an intro sentence to the topics covered in this particular section. Something along these lines:

"The following sections cover several key issues for users developing applications with the nRF70 Series ICs  using the respective development and evaluation boards."